src/base/base.scss

```diff
@@ -76,10 +76,43 @@ button {
 }
 
 details {
-  summary { cursor: pointer; }
+  summary {
+    cursor: pointer;
+  }
 
   &:not([open]) {
     // Set details content hidden by default for browsers that don't do this
-    > *:not(summary) { display: none !important; }
+    > *:not(summary) {
+      display: none !important;
+    }
+  }
+}
+
+// global focus styles
+
+// fallback :focus state
+:focus,
+.focus {
+  &:is(a, button, input, textarea, select) {
+    @include focusOutline;
+  }
+}
+
+// remove fallback :focus if :focus-visible is supported
+:focus:is(a, button, input, textarea, select):not(:focus-visible) {
+  outline: solid 1px transparent;
+}
+
+// default :focus-visibile state
+:focus-visible:is(a, button, input, textarea, select) {
```

Comment:
is summary needed in each of the “a, button, input, textarea, select” selectors?

Reply:
I'd prefer to target `summary` directly in the context of `details-menu` in case its ever used for something other than an interactive dropdown